block-storage-attacher/tests/e2e/e2e-tests/e2eTests.txt

```diff
@@ -0,0 +1,9 @@
+BlockVolumeAttacher-Volume-Test: Cluster-Creation: PASS
```

Comment:
Why is this file added?

Reply:
Also why is log file ` block-storage-attacher/tests/e2e/e2e-tests/log.txt` added?

Reply:
We will use this in future to provide the log file in case of failure